fe/src/components/Header/Header.tsx

```diff
@@ -44,7 +45,9 @@ export default function Header(): JSX.Element {
             ))}
           </HStack>
         </HStack>
-        <ColorMode />
+        <HStack spacing={12} ml="auto" title={colorMode === 'light' ? 'Switch to dark theme' : 'Switch to light theme'}>
```

Comment:
title should be inside Colormode.tsx component, because right now every element in HStack has this onHover title.